docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -193,8 +205,11 @@ You configure your {connectors} instance to listen for events from the data sour
 
 For this example, you create an instance of the Data Generator source connector. The Data Generator is provided for development and testing purposes. You specify the text for a message and how often to send the message.
 
-ifndef::qs[]
 .Prerequisites
+
+* If you want to use a dead letter queue (DLQ) to handle any messaging errors, create a Kafka topic to use for the DLQ.
```

Comment:
Now that you've added this DLQ text, you might want to tweak the existing text that refers to the topic for the **Dead letter queue** error handling policy (in both the Source and Sink sections)?